Orange/tests/test_contingency.py

```diff
@@ -39,8 +39,8 @@ def test_discrete(self):
 
     def test_discrete_missing(self):
         d = data.Table("zoo")
-        d.Y[25] = float("nan")
-        d[0][0] = float("nan")
+        d.loc[d.index[25], d.domain.class_var] = float("nan")
```

Comment:
This is not code. These are tests, made compatible with as little change as possible. The useful function code will always be much cleaner.

Ideally, this would be:
```py
d.type.iloc[25] = np.nan
```
But our current code makes it hard to adapt it so. I suppose it will have to get a bit more ugly before it gets neat.